examples/password_strength/README.md

```diff
@@ -0,0 +1,20 @@
+# 
```

Comment:
```suggestion
# Password Strength Estimator Example
```

Reply:
Also, please don't forget to update the [List of examples](https://github.com/yewstack/yew/tree/master/examples#list-of-examples) at the root of the examples directory.